src/MultiAgentReinforcementLearning/Project.toml

```diff
@@ -0,0 +1,13 @@
+name = "MultiAgentReinforcementLearning"
+uuid = "c65af49a-feda-45df-9d14-b4e40efca6f2"
+authors = ["Panajiotis Keßler <panajiotis@christoforidis.net>"]
+version = "0.1.0"
+
+[deps]
+BSON = "fbb218c0-5317-5bc6-957e-2ee96dd4b1f0"
+Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+ReinforcementLearningBase = "e575027e-6cd6-5018-9292-cdc6200d2b44"
```

Comment:
Can you add Compat bounds as appropriate?

Reply:
I am working on the newest versions of the Libraries. Shall I add them nonetheless?

Reply:
Yep! No need to pin to point versions, but at least to the minor release versions / ranges which are acceptable

Reply:
I did the compat entries